legend.go

```diff
@@ -40,6 +40,11 @@ type Legend struct {
 	// final position.
 	XOffs, YOffs vg.Length
 
+	// YPosition specify the vertical position of a legend entry.
+	// Valid values are [-1,+1], with +1 being the top of the
```

Comment:
This is not true from what I see in the code. It is in {-1,0,1}, rather than in [-1,+1]. I think it should match the documentation rather than changing the docs.